apps/me/.env.local.example

```
@@ -22,4 +22,5 @@ GCS_EMULATOR_HOST=localhost:9023
 
 # App
 NEXT_PUBLIC_SITE_URL=http://localhost:3003
+NEXT_PUBLIC_GA_MEASUREMENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GA_MEASUREMENT_ID key should go before the NEXT_PUBLIC_SITE_URL key

(UnorderedKey)